types/events.d.ts

```diff
@@ -0,0 +1,10 @@
+/// <reference types="node" />
+export default events;
+export function initializeEvents(): void;
+export function stopEvents(): void;
+/**
+ * Event emitter providing events from underlying Actor infrastructure and Apify package.
+ * @ignore
+ */
+declare const events: EventEmitter;
+import EventEmitter from "events";
```

Comment:
this one is tricky: 
![image](https://user-images.githubusercontent.com/461084/73302882-23670080-41f4-11ea-93d1-e2ec91334612.png)
![image](https://user-images.githubusercontent.com/461084/73302902-2e219580-41f4-11ea-8e45-debe06fb59de.png)
because of the triple slash comment at the top, it's having a hard time determining which one to use, guess either use the "import EventEmitter" for everything

Reply:
I remember, back in my days, we haven't had these problems. We had *Fully qualified identifiers*.

This causes tests to fail. I'm looking into it, but @mnmkng might have an insight for us:
```
 FAIL  test/utils.test.js
  ● utils.getMemoryInfo() › works WITHOUT child process outside the container

    expect(received).toMatchObject(expected)

    - Expected
    + Received

      Object {
    -   "childProcessesBytes": 0,
    +   "childProcessesBytes": 567042048,
        "freeBytes": 222,
        "totalBytes": 333,
        "usedBytes": 111,
      }

      143 |         try {
      144 |             const data = await Apify.getMemoryInfo();
    > 145 |             expect(data).toMatchObject({
          |                          ^
      146 |                 totalBytes: 333,
      147 |                 freeBytes: 222,
      148 |                 usedBytes: 111,

      at Object.test (test/utils.test.js:145:26)

  ● utils.getMemoryInfo() › works WITHOUT child process inside the container

    expect(received).toMatchObject(expected)

    - Expected
    + Received

      Object {
    -   "childProcessesBytes": 0,
    +   "childProcessesBytes": 567042048,
        "freeBytes": 222,
        "totalBytes": 333,
        "usedBytes": 111,
      }

      174 |         try {
      175 |             const data = await Apify.getMemoryInfo();
    > 176 |             expect(data).toMatchObject({
          |                          ^
      177 |                 totalBytes: 333,
      178 |                 freeBytes: 222,
      179 |                 usedBytes: 111,

      at Object.test (test/utils.test.js:176:26)
```

Reply:
I hate this test, it sometimes starts failing for reasons totally unknown and then mysteriously starts working again some moments later. I checked it multiple times and there doesn't seem to be anything that could cause it to be flaky, but hey, who knows. Unless its failing with each and every test run, just ignore it.

Reply:
On the other hand, this looks different than the typical failure. Anyway, as I said, unless it happens all the time, disregard it.

Reply:
@yin sadly Javascript is a bit (lot) more wild than Java and Rust 😶 the node types is a "best effort" on top of the commonjs that soon will be gone, hopefully (hence the triple declaration alias trying to cover all bases). if Node code was in TS, that would never happen. One class = one type. Clearly defined and exported interfaces / enums. 

`import * as EventEmitter from 'events'` is the non-es interop way of loading the event emitter (being exported as `module.exports = EventEmitter`). 
`import EventEmitter from 'events'` is a synthetic default import (doesn't actually exist in node code). 
`import { EventEmitter } from 'events'` is `EventEmitter.EventEmitter`, that is the same "class" referencing to itself as a static member, back from node 0.10. it's messy

https://github.com/nodejs/node/blob/c9b93e234454322ac0b7a6cd29d394f428f3e37d/lib/events.js#L68-L73

but if the tests are failing for using the "inner" EventEmitter, then `Apify.events` should be made as `any`, instead of breaking the compilation and shouldn't affect